library/ssl_tls13_generic.c

```diff
@@ -4485,6 +4497,9 @@ static int ssl_finished_in_postprocess( mbedtls_ssl_context* ssl )
 {
 	int ret = 0;
 
+#if defined(MBEDTLS_SSL_SRV_C)
```

Comment:
Readability can be improved here by returning early in case of a `ssl->conf->endpoint == MBEDTLS_SSL_IS_SERVER`.

Reply:
Done.